app/src/test/java/de/westnordost/streetcomplete/data/user/statistics/StatisticsParserTest.kt

```diff
@@ -24,7 +25,7 @@ class StatisticsParserTest {
             ),
             2345,
             78,
-            OffsetDateTime.parse("2007-12-03T10:15:30+01:00").toInstant().toEpochMilli(),
+            Instant.parse("2007-12-03T10:15:30+01:00").toEpochMilli(), //TODO doesn't support +HH:MM suffix? https://github.com/Kotlin/kotlinx-datetime#converting-instant-and-local-datetime-to-and-from-string
```

Comment:
The `lastUpdate` field in the to-be-parsed JSON is created here

https://github.com/streetcomplete/sc-statistics-service/blob/45d2d73a81e0941817ae343fe093f9a94802cdb9/get_statistics.php#L60

i.e. `date('c', $last_update)`.
In the [documentation](https://www.php.net/manual/en/datetime.format.php), this is described as:

<html><body>
<!--StartFragment-->

c | ISO 8601 date | 2004-02-12T15:19:21+00:00
-- | -- | --


<!--EndFragment-->
</body>
</html>

Your link mentions that only ISO-8601 is supported. So, I don't know... is one of the documentations incorrect?

Reply:
False alarm, it uses the same format (at least on JVM) https://github.com/streetcomplete/StreetComplete/pull/4471/commits/bea813ad99b66c6062db4d9cf103ef0e0f87e5a3